.gitignore

```diff
@@ -10,3 +10,12 @@ packages/*/*.js
 packages/*/bin
 !packages/*/.eslintrc.js
 *.log
+
+#TS output
+packages/**/dist
```

Comment:
This is only for the admin Argo, right? Will we be aligning the build pipelines?

Reply:
Yes, I'll be doing that in the next PR so that we build our packages the same way